api/src/pcapi/tasks/finance_tasks.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from pcapi import settings
+from pcapi.core.finance import models as finance_models
+from pcapi.routes.serialization import BaseModel
+from pcapi.tasks.cloud_task import list_tasks
+from pcapi.tasks.decorator import task
+
+
+logger = logging.getLogger(__name__)
+
+
+class GenerateInvoicesPayload(BaseModel):
+    batch_id: int
+
+
+@task(settings.GENERATE_INVOICES_QUEUE_NAME, "/finance/generate-invoices")
+def generate_invoices_task(payload: GenerateInvoicesPayload) -> None:
+    from pcapi.core.finance import api as finance_api
+
+    logger.info("Generate invoices from batch : %s", payload.batch_id)
+    batch = finance_models.CashflowBatch.query.get(payload.batch_id)
+
+    try:
+        finance_api.generate_invoices(batch)
+        # TODO: send mail about succesful generation
+    except Exception as exc:  # pylint: disable=broad-except
+        # TODO: send mail about failed generation
+        logger.warning("Invoices generation failed for batch : %s.", payload.batch_id, extra={"exception": exc})
```

Comment:
Avec `warning()`, on aura seulement l'exception levée. Mieux vaut utiliser `exception` qui nous donnera le traceback complet, fort utile pour débugger.

```suggestion
        logger.exception("Invoices generation failed for batch : %s.", payload.batch_id, extra={"exception": exc})
```